blocks/qanda/qanda.css

```diff
@@ -0,0 +1,104 @@
+/* nothing yet */
+
+.qanda > div {
+    @media (min-width: 992px) {
+        margin-left: 16.6667%;
+    }
+
+    @media (min-width: 992px) {
+        -webkit-box-flex: 0;
+        flex: 0 0 66.6667%;
+        max-width: 66.6667%;
+    }
+
+    position: relative;
+    width: 100%;
+    padding-right: 20px;
+    padding-left: 20px;
+    background-color: transparent;
+    margin: 3rem auto 5rem;
+}
+
+.qanda-content > div {
+    width: 100%;
+    overflow: hidden;
+    padding: 0;
+    display: block;
+    border-bottom: 1px solid #e5e7fa;
+}
+
+.qanda-content > div::after {
+    background-color: #dcdde6;
+    border: 0;
+    height: .1rem;
+    margin-bottom: 3rem;
+    margin-top: 3rem
+}
+
+.qanda-content > div > label {
+    display: -webkit-box;
```

Comment:
the `-webkit-` prefixes are usually not needed. Please try without and test it.

Reply:
removed